File: CodingBat/Python/Warmup_2/string_splosion.py
#Given a non-empty string like "Code return a string like CCoCodCode"
#The splosion is copying the string over on it self, the number of times equal to the length of the original string. With the number of characters being equal to the number of the iteration
def string_splosion(str):
    x=""
    for i in range(len(str)):
       x+=str[:i+1]
    return x

File: CodingBat/Java/Warmup_2/stringSplosion.java
public class stringSplosion{
    
    public static String stringSplosion(String str){
        // Given a non-empty string like "Code" return a string like "CCoCodCode".
        if(str.length()<=1){
            return str;
        }
        else{
            String str2="";
            for(int i=0;i<str.length();i++){
                str2+=str.substring(0,i);
            }
            return str2+str;
        }
    }
    
    public static void main(String[] args) {
        System.out.println(stringSplosion("Code"));
    }
}